src/sentry/mediators/external_requests/alert_rule_action_requester.py

```diff
@@ -41,8 +41,9 @@ def _make_request(self):
                 method=self.http_method,
                 data=self.body,
             )
-            body = safe_urlread(req)
-            response = {"success": True, "message": "", "body": json.loads(body)}
+            body = safe_urlread(req).decode() if safe_urlread(req) else None
```

Comment:
The name `safe_urlread` sounds like it had side effects.
```py
def safe_urlread(response):
    return response.content
```
Like if `response` is a stream object or something you might only be able to efficiently call `.content` once.

I'd do: 
```py
def _make_request(self) -> Mapping[str, bool | str]:
    try:
        req = ...
    except Exception as e:
        ...
        return {"success": False, "message": message}

    body_raw = safe_urlread(req)
    body = body_raw.decode() if body_raw else None
    message = f"{self.sentry_app.name}: {body or SUCCESS_MESSAGE}"
    return {"success": True, "message": message}
```
So that the try/catch is also only on one line.